vision/api/face_detection/faces_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import shutil
+import tempfile
+
+import pytest
+from PIL import Image
+from faces import main
+
+
+@pytest.yield_fixture
```

Comment:
pytest has this built-in: https://pytest.org/latest/builtin.html#builtin-fixtures-function-arguments